src/components/comments/index.test.ts

```diff
@@ -0,0 +1,41 @@
+import { CommentsComponent } from './index';
+
+describe('CommentsComponent', () => {
+  let commentsComponent: CommentsComponent;
+
+  beforeEach(() => {
+    commentsComponent = new CommentsComponent();
+  });
+
+  it('should create a new instance of CommentsComponent', () => {
+    expect(commentsComponent).toBeInstanceOf(CommentsComponent);
+  });
+
+  it('should have a name property', () => {
+    expect((commentsComponent as any).name).toEqual('Comments');
+  });
+
+  it('should have a logger property', () => {
+    expect((commentsComponent as any).logger).toBeDefined();
+  });
+
+  it('should have an element property', () => {
+    expect((commentsComponent as any).element).toBeUndefined();
+  });
+
+  it('should create a new element when start() is called', () => {
+    (commentsComponent as any).start();
+    expect((commentsComponent as any).element).toBeDefined();
+  });
+
+  it('should add the element to the document body when start() is called', () => {
+    (commentsComponent as any).start();
+    expect(document.body.contains((commentsComponent as any).element)).toBe(true);
+  });
+
+  it('should remove the element from the document body when destroy() is called', () => {
+    (commentsComponent as any).start();
+    (commentsComponent as any).destroy();
+    expect(document.body.contains((commentsComponent as any).element)).toBe(false);
+  });
```

Comment:
It is generally bad practice to call private methods directly in tests.

Private methods must be tested by public methods and properties, in which case you can call attach and detach, which are public in the abstract class, instead of start and destroy.

```suggestion
  it('should remove the element from the document body when destroy() is called', () => {
    commentsComponent.attach({
    realtime: ABLY_REALTIME_MOCK,
      localParticipant: MOCK_LOCAL_PARTICIPANT,
      group: MOCK_GROUP,
      config: MOCK_CONFIG,
      });
    commentsComponent.detach();
    expect(document.body.contains((commentsComponent as any).element)).toBe(false);
  });
```